index.html

```diff
@@ -1,1115 +1,1032 @@
+        </div>
+        <div class="carousel-item">
+          <img src="./images/carosul4.jpg" class="d-block w-100" alt="...">
+        </div>
+        <div class="carousel-item">
+          <img src="./images/carosul5.jpg" class="d-block w-100" alt="...">
+        </div>
+        <div class="carousel-item">
+          <img src="./images/carosul6.png" class="d-block w-100" alt="...">
+        </div>
+        <div class="carousel-item">
+          <img src="./images/carosul7.jpg" class="d-block w-100" alt="...">
+        </div>
+        <div class="carousel-item">
+          <img src="./images/carosul8.jpg" class="d-block w-100" alt="...">
+        </div>
+      </div>
+      <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleControls"
+        data-bs-slide="prev">
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="visually-hidden">Previous</span>
+      </button>
+      <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls"
+        data-bs-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="visually-hidden">Next</span>
+      </button>
+    </div>
+  </section>
+
+
+
+
+
+  <section class="themes" id="themes">
+    <h2 class="heading-theme">
+      <span class="heading-theme-1">THEMES</span>
+    </h2>
+    <div class="themes-cards">
+      <div class="theme-card blockchain">
+        <div class="card-title">
+          BLOCKCHAIN
+        </div>
+      </div>
+      <div class="theme-card ar-vr">
+        <div class="card-title">
+          AR/VR
+        </div>
+      </div>
+      <div class="theme-card fintech">
+        <div class="card-title">
+          FINTECH
+        </div>
+      </div>
+      <div class="theme-card drones">
+        <div class="card-title">
+          AUTONOMOUS ROBOTICS & DRONES
+        </div>
+      </div>
+      <div class="theme-card health-ed">
+        <div class="card-title">
+          HEALTH & EDUCATION
+        </div>
+      </div>
+      <div class="theme-card open-innovation">
+        <div class="card-title">
+          OPEN INNOVATION
+        </div>
+      </div>
+    </div>
+
+
+  </section>
+
+
+
+  <section id="prizes" class="prizes">
+    <div class="container">
+      <div class="sec-title title-prizes-div">
+        <h2 class="title-prizes" style="color: #fff;">PRIZES</h2>
+      </div>
+      <div class="prize-cards">
+        <div class="prize-card first-prize">
+          <div class="prize-card-content">
+            COMING SOON
+          </div>
+        </div>
+        <div class="prize-card second-prize">
+          <div class="prize-card-content">
+            COMING SOON
+          </div>
+        </div>
+        <div class="prize-card third-prize">
+          <div class="prize-card-content">
+            COMING SOON
+          </div>
+        </div>

```

Comment:
Please don't make PR against `master` if the code is incomplete. Instead merge the changes to a new dev branch or resume working on the existing branch until the changes are final, and then make a PR against `master`.